README.md

```diff
@@ -1,16 +1,19 @@
 Developer guide to Tessel 2 Docs Gitbook
 ============
 
-# Tools
-Install the following command line tools using `npm i {{toolname}} -g`
+# Requirements
+The docs are build using a command line tool called [GitBook](https://www.npmjs.com/package/gitbook), which requires:
 
-* [gitbook](https://www.npmjs.com/package/gitbook)
```

Comment:
Because of the need for global install, still need to call these out in the instructions